packages/flutter/lib/src/rendering/error.dart

```diff
@@ -104,6 +109,11 @@ class RenderErrorBox extends RenderBox {
           width = size.width;
         }
         _paragraph.layout(new ui.ParagraphConstraints(width: width));
+
+        // Heuristically offset the top of the text to prevent system status bar overlap if the
+        // widget is tall enough and likely to overlap.
+        if (size.height > 3 * systemTopPadding)
```

Comment:
If we do the three-line thing we don't have to do anyhting with the padding.

I'm not comfortable with us trying to read "size" here. This object is not in a sane state if it's being rendered.